<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_TagEdit</class>
 <widget class="QWidget" name="GUI_TagEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>346</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>20000</width>
    <height>20000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ID3 Tagger</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="lab_title">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Album</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Artist</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Genre</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string># Track</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="9">
    <widget class="QLineEdit" name="le_title"/>
   </item>
   <item row="4" column="1" colspan="9">
    <widget class="QLineEdit" name="le_album"/>
   </item>
   <item row="5" column="1" colspan="9">
    <widget class="QLineEdit" name="le_artist"/>
   </item>
   <item row="6" column="1" colspan="9">
    <widget class="QLineEdit" name="le_genre"/>
   </item>
   <item row="4" column="10">
    <widget class="QCheckBox" name="btn_all_album">
     <property name="text">
      <string>all</string>
     </property>
    </widget>
   </item>
   <item row="5" column="10">
    <widget class="QCheckBox" name="btn_all_artist">
     <property name="text">
      <string>all</string>
     </property>
    </widget>
   </item>
   <item row="6" column="10">
    <widget class="QCheckBox" name="btn_all_genre">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>all</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="6">
    <widget class="QProgressBar" name="pb_progress">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="7" column="10">
    <widget class="QCheckBox" name="btn_all_year">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>all</string>
     </property>
    </widget>
   </item>
   <item row="7" column="7" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Year</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="9">
    <widget class="QSpinBox" name="sb_year">
     <property name="maximumSize">
      <size>
       <width>1564654</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>2100</number>
     </property>
     <property name="value">
      <number>2011</number>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="8" colspan="3">
    <widget class="QDialogButtonBox" name="bb_ok_cancel">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="9">
    <widget class="QLineEdit" name="le_comment"/>
   </item>
   <item row="9" column="8">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="sb_track_num"/>
   </item>
   <item row="1" column="1" colspan="9">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_prev">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_track_num">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Track 1/15</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_next_track">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="9">
    <widget class="QLineEdit" name="lab_filepath">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="10" rowspan="2">
    <widget class="QLabel" name="lab_icon">
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
